### PR TITLE
Run as root and update docs

### PR DESCRIPTION
**Pull request by AMecea**
FIXES ENG-470

### CODE REVIEW

**Comment by rstefan1** [comment]
Doesn't work on Linux. Cannot upload media:

image

**Comment by rstefan1** [comment]
```
→ docker compose exec wordpress bash
root@585f0e8ad8c8:/app# wp cache flush
Error: YIKES! It looks like you're running this as root. You probably meant to run this as the user that your WordPress installation exists under.

If you REALLY mean to run this as root, we won't stop you, but just bear in mind that any code on this site will then have full control of your server, making it quite DANGEROUS.

If you'd like to continue as root, please run this again, adding this flag:  --allow-root

If you'd like to run it as the user that this site is under, you can run the following to become the respective user:

    sudo -u USER -i -- wp <command>

```

File: docker-compose.yaml [hunk 1]
```diff
@@ -2,8 +2,9 @@ version: "3.3"
 
 services:
   wordpress:
-    image: docker.io/presslabs/wordpress-runtime:latest-wp-php-8.1
-    user: "${USER_ID:-1000}:${GROUP_ID:-1000}"
+    #image: docker.io/presslabs/wordpress-runtime:latest-wp-php-8.1
+    image: docker.io/presslabs/wordpress-runtime:classic-6.2-php-8.1.17-update-lua-bitpoke
```
Comment (rstefan1) [inline docker-compose.yaml:6]:
```suggestion
    image: docker.io/presslabs/wordpress-runtime:latest-wp-php-8.1
```

**Comment by rstefan1** [comment]
> ```
> → docker compose exec wordpress bash
> root@585f0e8ad8c8:/app# wp cache flush
> Error: YIKES! It looks like you're running this as root. You probably meant to run this as the user that your WordPress installation exists under.
> 
> If you REALLY mean to run this as root, we won't stop you, but just bear in mind that any code on this site will then have full control of your server, making it quite DANGEROUS.
> 
> If you'd like to continue as root, please run this again, adding this flag:  --allow-root
> 
> If you'd like to run it as the user that this site is under, you can run the following to become the respective user:
> 
>     sudo -u USER -i -- wp <command>
> ```

We can use: `docker compose exec -u www-data wordpress bash`

**Comment by rstefan1** [comment]
> > ```
> > → docker compose exec wordpress bash
> > root@585f0e8ad8c8:/app# wp cache flush
> > Error: YIKES! It looks like you're running this as root. You probably meant to run this as the user that your WordPress installation exists under.
> > 
> > If you REALLY mean to run this as root, we won't stop you, but just bear in mind that any code on this site will then have full control of your server, making it quite DANGEROUS.
> > 
> > If you'd like to continue as root, please run this again, adding this flag:  --allow-root
> > 
> > If you'd like to run it as the user that this site is under, you can run the following to become the respective user:
> > 
> >     sudo -u USER -i -- wp <command>
> > ```
> 
> We can use: `docker compose exec -u www-data wordpress bash`

Works with:

```
USER_ID="$(id -u)" GROUP_ID="$(id -g)" docker compose up
docker compose exec -u www-data wordpress bash

**Comment by rstefan1** [comment]
> > ```
> > → docker compose exec wordpress bash
> > root@585f0e8ad8c8:/app# wp cache flush
> > Error: YIKES! It looks like you're running this as root. You probably meant to run this as the user that your WordPress installation exists under.
> > 
> > If you REALLY mean to run this as root, we won't stop you, but just bear in mind that any code on this site will then have full control of your server, making it quite DANGEROUS.
> > 
> > If you'd like to continue as root, please run this again, adding this flag:  --allow-root
> > 
> > If you'd like to run it as the user that this site is under, you can run the following to become the respective user:
> > 
> >     sudo -u USER -i -- wp <command>
> > ```
> 
> We can use: `docker compose exec -u www-data wordpress bash`

Fixed. Works with:

```
USER_ID="$(id -u)" GROUP_ID="$(id -g)" docker compose up
docker compose exec wordpress bash

I have no name!@c5f92be365fa:/app$ wp cache flush
PHP Warning:  Undefined array key "HTTP_HOST" in phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Runner.php(1296) : eval()'d code on line 89
Success: The cache was flushed.
I have no name!@c5f92be365fa:/app$ 
```

File: README.md [hunk 2]
```diff
@@ -35,28 +35,36 @@ With Docker installed and started run these commands from your terminal window
...
 * obtain a database dump from your hosting provider
 * download it locally and unpack it
-* copy it to the Docker DB container:
+* import the database into local MySQL:
 
-    `docker cp database.sql gitium-base-db-1:/database.sql`
+    cat database.sql | docker compose exec -T db mysql -u wordpress -pnot-so-secure wordpress
 
-* connect to the MySQL terminal either from Docker Desktop, or by running:
+* connect to the wordpress-1 container:
```
Comment (rstefan1) [inline README.md:42]:
```suggestion
* connect to the wordpress container:
```

**Comment by rstefan1** [comment]
> Doesn't work on Linux. Cannot upload media:
> 
> image

Fixed.

**Comment by rstefan1** [comment]
@AMecea cannot update wp:

image

**Comment by rstefan1** [comment]
Tests on Linux:
- [ ] upload an image
- [ ] install plugin

**Review by rstefan1 — APPROVED** [review]

**Review by pdobrescu — APPROVED** [review]
LGTM

**Review by rstefan1 — APPROVED** [review]

**Pull request merged**